fix(Core/Spells): Ensures that procs triggered by another aura are pr…

UltraNix opened the pull request:
…occessed in the first place.

Fixed #4720.

<!-- First of all, THANK YOU for your contribution.
 Please fill this template and do not forget to have a look at our Pull Request tutorial: https://www.azerothcore.org/wiki/How-to-create-a-PR
-->


## Changes Proposed:
-  Aura procs triggered by another proc aura should be proccessed in the first place.
-  


## Issues Addressed:
- Closes #4720.
<!-- If the issue does not exist, please describe it and how to reproduce it. If the issue already exists, just paste the link to the issue you close, like this: Closes https://github.com/azerothcore/azerothcore-wotlk/issues/967 -->


## SOURCE:
<!-- If this pull request IS linked with in-game content, please include any evidence/documentation/video or further proof in order to guarantee that the proposed changes described above are the correct ones.
 - If it is described in a guide/post or Wowhead comment, please include the link.
 - Can you link a video that confirms it?
 - Please share the source which states how it should work.
 - If this pull request IS NOT linked with in-game content, please leave this field as N/A
-->


## Tests Performed:
<!-- Does it build without errors? Did you test in-game? What did you test? Did you do all these tests on Linux/Mac/Windows? Describe any other tests performed -->
- Tested ingame with Rogue's Flurry.
- 


## How to Test the Changes:
<!-- We need to confirm the changes are going to be working, so please describe in general and for non-developers how to test the changes:
 - Which commands to use? Which NPC to teleport to?
 - Do we need to enable debug flags on Cmake?
 - Do we need to look at the console?
 - Describe any other steps
-->


## Known Issues and TODO List:
<!-- This is a TODO list with checkboxes to tick -->
- [ ]
- [ ] 


## Target Branch(es):
- [x] Master


<!-- NOTES: 
 - You do not need to squash your commits, on merge, we will squash them for you (when there are too many commits we merge them into one big commit for a cleaner and easy-to-read history).
 - If you intend to contribute repeatedly to our project, it is a good idea to join our discord channel. We set ranks for our contributors and give them access to special resources or knowledge: https://discord.com/invite/DasJqPba -->



<!-- Do not remove the instructions below about testing, they will help users to test your PR -->
 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

UltraNix commented:
@Stifler82 Please, test it.

Kitzunu commented on src/server/game/Entities/Unit/Unit.cpp:
```diff
@@ -5855,6 +5855,7 @@ void Unit::SendPeriodicAuraLog(SpellPeriodicAuraLogInfo* pInfo)
                     damage = 0;
                 }
 
+
```
remove pls

UltraNix commented:
@Kitzunu Done.

Stifler82 commented:
@UltraNix 
If i read good the Fix, this isn't a specifici Fix, for the specific issue, right?
This is a Fix that make work like it intended all Spells that work like Flurry, right?(which should be all spells with Stacks) right?
so this is a "Generic Fix" right?

UltraNix commented:
This is a generic fix for spells that triggers another spell and both can proc at the same time, as e.g. Flurry. In that case, the triggered spell should be handled first in order to properly "eat" stacks/charges before the "main" spell proc it again.

Stifler82 commented:
> This is a generic fix for spells that triggers another spell and both can proc at the same time, as e.g. Flurry. In that case, the triggered spell should be handled first in order to properly "eat" stacks/charges before the "main" spell proc it again.

@UltraNix 
Ok, maybe I explained it badly, (I apologize), but yes, I understood well then..
But tell me, and if there is a spell that works in Stack, (BUT!) It works slightly different and it doesn't have to reach (The maximum number of Stacks) with this generic fix don't we bugger that spell?
(now I don't have a practical example at hand, because right now they don't come to mind)..

UltraNix commented:
So you have to tell me that example so I can look at it. For now, I don't see any.

Stifler82 commented:
> So you have to tell me that example so I can look at it. For now, I don't see any.

Warning: I'm not criticizing your fix, on the contrary, I think it's a Smart Fix.
But I wondered, and if there is a spell that works different, I would not want to fix a spell, and then we bugged others 10

UltraNix commented:
Then again: test it and give me feedback about bugged spell. I create fixes, not bugs (not on purpose). I always check any influnce of my fixes on the core. Wondering or maybe "theorizing" about potential bugs is a waste of time. Just test it.

Stifler82 commented:
> Then again: test it and give me feedback about bugged spell. I create fixes, not bugs (not on purpose). I always check any influnce of my fixes on the core. Wondering or maybe "theorizing" about potential bugs is a waste of time. Just test it.

@UltraNix 
Ok, i'm going to test it, and I let you know..

Stifler82 commented:
@UltraNix 
Ok, tested, the Fix works perfectly.
For me it can be merged (As for the Flurry talent).
Then, as said before, I don't know if this **"Generic" Fix** will lead to side effects, (in other Spells / talents).
**Video Test:** [Flurry Fixed](https://youtu.be/1Dzn_LvKf0I)

Stifler82 commented:
@UltraNix 
Now, having solved this problem, there is another question, or perhaps better to say, **"Dilemma"**, which is not directly related to this fix, but is related to talent.

it's all explained here: [Flurry Dilemma](https://youtu.be/uYI78Y4b3sk)
more than explained it is "Illustrated" here...

UltraNix commented:
`Wrath of the Lich King Patch 3.1.0 (2009-04-14): Speed increase changed to 5/10/15/20/25% (down from 10/15/20/25/30%)`
I don't see any bug here. Blizzard many times forgot to update spell tooltips.  It should be 5% per talent point and this value comes directly from Blizz files.

Stifler82 commented:
@UltraNix 
What I showed you, is NOT the TOOLTIP!
what I have shown you, are the TALENTS that are WRONG!

UltraNix commented:
Explain to me where is the bug? Maybe I misunderstood you.

FrancescoBorzi commented:
@Stifler82 can this PR be merged?

Stifler82 commented:
@UltraNix 
Video: [Flurry 3](https://youtu.be/2gcvkZiHL44)

UltraNix commented:
@Stifler82 Now I got it :)

UltraNix commented:
https://wowpedia.fandom.com/wiki/Flurry_(warrior_ability)
https://wowpedia.fandom.com/wiki/Flurry_(shaman_ability) 
For warriors values changes with TBC patch and for shaman in wotlk? Weird.

Stifler82 commented:
> @Stifler82 Now I got it :)
> So the talent's tooltip says 30%, aura with 30% is applied and Wowpedia/wow patches says it should be 25%. Something is wrong here.

No, wowpedia But Blizzard's WOTLK Patch Notes..

https://www.toxicgaming.us/wow/wrath-patch-notes/

Stifler82 commented:
> https://wowpedia.fandom.com/wiki/Flurry_(warrior_ability)
> https://wowpedia.fandom.com/wiki/Flurry_(shaman_ability)
> For warriors values changes with TBC patch and for shaman in wotlk? Weird.

I don't understand the comparison you are making, one is the talent of the shaman, the other is the talent of the warrior.

They have the same name, the same function, but they are 2 different things, but above all 2 different IDs.

UltraNix commented:
Because they are the same effect and the same changes has been done.

Stifler82 commented:
> Because they are the same effect and the same changes has been done.

They simply made an arbitrary choice, without any particular logic, they first changed the talent of the warrior, then maybe it fit well for them as they changed the talent of the warrior in TBC and later in WOTLK they did the same with the talent of the Shaman, so what's Weird?

UltraNix commented:
Maybe you are right that they decided to change that to shaman after seeing results with warrior.

Stifler82 commented:
I made another video for you where I clarify an important point that I'm not sure is understood in the last video.
**Video:** [Flurry 4](https://youtu.be/oCn2cUZm44w)

UltraNix commented:
@Stifler82 Test it now. Changed values serverside. But ingame tooltips are hardcoded in client, so I can't change them.

Stifler82 commented:
> @Stifler82 Test it now. Changed values serverside. But ingame tooltips are hardcoded in client, so I can't change them.

i can fix it, don't worry👍

Stifler82 reviewed (changes requested):
@UltraNix 
I don't know, maybe I explained myself badly..
You put the wrong ID's spell..

you have to correct it in this way:
`ApplySpellFix({ 12319, 12971, 12972, 12973, 12974 }, [](SpellEntry* spellInfo)`

UltraNix commented:
Nope. `12319, 12971, 12972, 12973, 12974` are proc auras that trigger the ones I corrected.

UltraNix commented:
![1 0](https://user-images.githubusercontent.com/80540499/113340480-3d3a3100-932c-11eb-8cba-2521b32d7449.png)
![1 1](https://user-images.githubusercontent.com/80540499/113340474-3c090400-932c-11eb-9cdf-f809a9851124.png)

Stifler82 commented:
**Do not merge this fix yet!!!**
-
there have been errors both from Me and from UltraNix, now I am too tired and I cannot explain, tomorrow when I wake up, (since I have been awake for 48h+) we will discuss it better.

UltraNix commented:
If I understand correctly:
The problem was with Shaman's Flurry and its ranks -  was 6/12/18/24/30%, instead of 5/10/15/20/25%, just like the version for warriors. So I changed values and now they are same as Warrior's flurry - 5/10/15/20/25%.
`12319, 12971, 12972, 12973, 12974` - these IDs are warriors' Flurry talents, so I needed the Shamans' ones: `16256, 16281, 16282, 16283, 16284`. These are only proc triggered auras, so I fixed the triggered auras - `16257, 16277, 16278, 16279, 16280`.
To verify it, test ingame and once the Flurry proc - type `.list auras` and see the Flurry proced aura value (or look at weapon speed in char sheet).

Stifler82 commented:
So @UltraNix, we made a bit of mess, but I know how to fix everything, all you have to do is:
Cancel this fix: `src/server/game/Spells/SpellMgr.cpp`
And just leave this fix: `src/server/game/Entities/Unit/Unit.cpp`

In a while I will post here the fix to be associated with this fix: `src/server/game/Spells/SpellMgr.cpp`
ok?

UltraNix commented:
But changes in `SpellMgr.cpp` are right or no? If so, what is wrong there?

Stifler82 commented:
> But changes in `SpellMgr.cpp` are right or no?

- Yes `SpellMgr.cpp`

>If so, what is wrong there?

- I'm working on it right now, and I'll explain you better when I'll finish it for make it clear to you too, i can't explain it now without a practice demonstration.

Stifler82 commented:
@UltraNix 
This is the fix you need to add (in addition to yours), with this fix all Talent Ranks will work at the correct percentages.

```
DELETE FROM `spell_dbc` WHERE (`ID` IN (16256, 16257, 16277, 16278, 16279, 16280, 16281, 16282, 16283, 16284));
INSERT INTO `spell_dbc` VALUES
(16256, 0, 0, 0, 208, 0, 0, 67108864, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 0, 0, 0, 1, 21, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16257, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 1','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16257s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190,'','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16257, 0, 0, 0, 0, 0, 0, 524288, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 3, 0, 0, 1, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 138, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2759, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 1','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16257s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Attack speed increased by $16257s1%.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 11, 0, 512, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16277, 0, 0, 0, 0, 0, 0, 524288, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 3, 0, 0, 1, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 9, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 138, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2759, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 2','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16277s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Attack speed increased by $16277s1%.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 11, 0, 512, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16278, 0, 0, 0, 0, 0, 0, 524288, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 3, 0, 0, 1, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 14, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 138, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2759, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 3','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16278s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Attack speed increased by $16278s1%.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 11, 0, 512, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16279, 0, 0, 0, 0, 0, 0, 524288, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 3, 0, 0, 1, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 19, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 138, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2759, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 4','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16279s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Attack speed increased by $16279s1%.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 11, 0, 512, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16280, 0, 0, 0, 0, 0, 0, 524288, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 3, 0, 0, 1, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 24, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 138, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2759, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 5','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16280s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Attack speed increased by $16280s1%.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 11, 0, 512, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16281, 0, 0, 0, 208, 0, 0, 67108864, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 0, 0, 0, 1, 21, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16277, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 2','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16277s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190,'','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16282, 0, 0, 0, 208, 0, 0, 67108864, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 0, 0, 0, 1, 21, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16278, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 3','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16278s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190,'','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16283, 0, 0, 0, 208, 0, 0, 67108864, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 0, 0, 0, 1, 21, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16279, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 4','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16279s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190,'','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0),
(16284, 0, 0, 0, 208, 0, 0, 67108864, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 69652, 100, 0, 0, 0, 1, 21, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16280, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 108, 0, 50, 'Flurry','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Rank 5','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 'Increases your attack speed by $16280s1% for your next 3 swings after dealing a critical strike.','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190,'','','','','','','','','', 0, 0, 0, 0, 0, 0, 0, 16712190, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0);
```

UltraNix commented:
Are you overriding existing spells? What differences between these entries and from dbc?

Stifler82 commented:
> Are you overriding existing spells? What differences between these entries and from dbc?

Login on Discord

UltraNix commented:
@FrancescoBorzi This can be merged as it fixed the addressed issue.
@Stifler82 is going to fix another problem with this spell out of scope this PR.

Stifler82 commented:
it can be merged.
@FrancescoBorzi @Helias

Stifler82 reviewed (approved):
Tested
